plugins/reporters/asciidoc/src/main/resources/templates/freemarker_implicit.ftl

```diff
@@ -6,6 +6,6 @@
 [#-- @ftlvariable name="ortResult" type="org.ossreviewtoolkit.model.OrtResult" --]
 [#-- @ftlvariable name="licenseTextProvider" type="org.ossreviewtoolkit.reporter.LicenseTextProvider" --]
 [#-- @ftlvariable name="LicenseView" type="org.ossreviewtoolkit.model.licenses.LicenseView" --]
-[#-- @ftlvariable name="helper" type="org.ossreviewtoolkit.reporter.utils.FreemarkerTemplateProcessor.TemplateHelper" --]
+[#-- @ftlvariable name="helper" type="org.ossreviewtoolkit.plugins.reporters.freemarker.FreemarkerTemplateProcessor.TemplateHelper" --]
```

Comment:
Could you please fix the `Set`s of `org.ossreviewtoolkit.reporter.utils.FreemarkerTemplateProcessor.PackageModel` as well?

Reply:
And maybe say in the commit message that this is a fixup for af2a476a301aaf29e317e91f08370f675c52ec79. Thanks!